nixos/tests/incus/default.nix

```diff
@@ -44,6 +44,12 @@ in
 
   zfs = incusTest {
     inherit lts pkgs system;
-    storageLvm = true;
+    storageZfs = true;
+  };
+
+  appArmor = incusTest {
+    inherit lts pkgs system;
+    appArmor = true;
+    instanceContainer = true;
```

Comment:
Let's reduce the scope a bit more and disable legacy init for now. `initLegacy = false;` will also make the tests quicker by reducing the number of nixos instances.